apps/console/src/features/identity-providers/components/forms/authenticators/microsoft-authenticator-form.tsx

```diff
@@ -430,32 +438,62 @@ export const MicrosoftAuthenticatorForm: FunctionComponent<MicrosoftAuthenticato
                 width={ 16 }
                 data-testid={ `${ testId }-authorized-redirect-url` }
             />
-            <Field.Input
+            <Field.Scopes
+                ariaLabel="Scopes provided by Microsoft Authenticator"
+                inputType="default"
+                name="Scopes"
```

Comment:
Lets add i18n placeholders here.
```suggestion
                name="Scopes"
```

Reply:
Fixed with 51ded5bd3ca56d4ec2c9f85a6b73b30573767c72